source/common/http/websocket/BUILD

```diff
@@ -24,8 +24,6 @@ envoy_cc_library(
         "//source/common/http:codes_lib",
         "//source/common/http:header_map_lib",
         "//source/common/http/http1:codec_lib",
-        # TODO(mattklein123): Common code reaching into extensions is not right. Sort this out
-        # when the HTTP connection manager is moved.
```

Comment:
Is this implicitly sorted out by documenting that tcp proxy and http connection mananger are special cased?